auth/api/bin/install.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+set -ex
+
+BASEDIR=$(dirname $0)
+
+$BASEDIR/console rabbitmq:setup-fabric
+$BASEDIR/console doctrine:database:create --if-not-exists
+$BASEDIR/console doctrine:schema:update -f
+$BASEDIR/console app:create-client ${AUTH_CLIENT_ID} \
+    --random-id=${AUTH_CLIENT_RANDOM_ID} \
+    --secret=${AUTH_CLIENT_SECRET} \
+    --grant-type password \
+    --grant-type authorization_code \
+    --redirect-uri ${AUTH_BASE_URL} \
+    --redirect-uri ${UPLOADER_BASE_URL} \
+    --redirect-uri ${EXPOSE_BASE_URL}
+$BASEDIR/console app:create-client ${UPLOADER_CLIENT_ID} \
```

Comment:
Auth is a central service which does know anything about satellite services (expose, uploader...).
The only thing the container could do is to create its own database schema but not adding service OAuth credentials.